vevestaX/vevesta.py

```diff
@@ -154,7 +154,10 @@ def endModelling(self):
         temp = dict(inspect.getmembers(inspect.stack()[1][0]))['f_locals'].copy()
         self.temp = inspect.getmembers(inspect.stack()[1])
 
-        self.__variables = {**self.__variables, **{i: temp.get(i) for i in temp if i not in self.__startlocals and i[0] != '_' and (type(temp[i]) in self.__primitiveDataTypes or isinstance(temp[i], (str, int, float, bool)))}}
```

Comment:
Make sure u check the git checkin your self and dont make unnecessary changes..not acceptable